state-synchronizer/src/chunk_request.rs

```diff
@@ -39,13 +39,24 @@ pub enum TargetType {
     Waypoint(Version),
 }
 
+impl TargetType {
+    pub fn version(&self) -> Option<u64> {
+        match self {
+            TargetType::TargetLedgerInfo(li) => Some(li.ledger_info().version()),
+            TargetType::HighestAvailable { target_li, .. } => {
+                target_li.as_ref().map(|li| li.ledger_info().version())
+            }
+            TargetType::Waypoint(version) => Some(*version),
+        }
+    }
+}
+
 impl fmt::Debug for TargetType {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(f, "{}", self)
     }
 }
 
-// TODO: This cuts out LedgerInfo, do we need it logging?
```

Comment:
So, the reason I add this TODO, is `LedgerInfo` is a huge object last I checked.  I would think about whether we need the entire thing or not.

Reply:
the Display format of ledger info with sigs omits the signatures, and I think that might not be that big? last I checked we wouldn't really need the signatures for debugging

Reply:
I guess the question is what are we looking for? ledger info itself has a lot of info only relevant to consensus, perhaps not critical but we should probably make a to_client_string or something like that such that we only show version and hash maybe?